index.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html lang="en">
+
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+
+		<title>Anna Russett</title>
+	
+		<link rel="icon" type="image/x-icon" href="http://www. /favicon.ico">
```

Comment:
lol i did this as a reminder to fill it in when i have a favicon to use and then forgot abt it n___n; is a favicon something we should ask sophia for?